gke-disk-image-builder/imager.go

```diff
@@ -149,6 +150,20 @@ func GenerateDiskImage(ctx context.Context, req Request) error {
 						Instance: compute.Instance{
 							Name:        fmt.Sprintf("%s-instance", req.JobName),
 							MachineType: fmt.Sprintf("zones/%s/machineTypes/%s", req.Zone, req.MachineType),
+							ServiceAccounts: []*compute.ServiceAccount{
+								&compute.ServiceAccount{
+									Email: req.ServiceAccount,
+									Scopes: []string{
+										"https://www.googleapis.com/auth/devstorage.read_only",
+										"https://www.googleapis.com/auth/logging.write",
+										"https://www.googleapis.com/auth/monitoring.write",
+										"https://www.googleapis.com/auth/pubsub",
```

Comment:
Look good! Btw, do we really need pubsub?

Reply:
Simplified to just use cloud-platform